src/benchmarks/micro/libraries/Microsoft.Extensions.Http/LoggingOverheadBenchmark.cs

```diff
@@ -0,0 +1,77 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using MicroBenchmarks;
+
+namespace Microsoft.Extensions.Http.Performance
+{
+    [BenchmarkCategory(Categories.Libraries)]
+    public class LoggingOverheadBenchmark
```

Comment:
I am not convinced that we should have these benchmarks here.

They do some work using fakes|mocks with logging enabled and disabled to measure the "logging overhead". IMHO we have enough of other logging benchmarks here to measure the logging performance and catch any logging regressions introduced in the future. If we ever want to measure logging performance we should profile some real-world ASP.NET Core app and analyze the profile data.

@maryamariyan could you please remove this class?